dashboard/app/views/pd/regional_partner_contact_mailer/_form_body.haml

```diff
@@ -0,0 +1,41 @@
+%p
+  %b Name:
+  - if @form[:title]
+    = "#{@form[:title]}" + ' '
+  = "#{@form[:first_name]}" + ' ' + "#{@form[:last_name]}"
+  %br
+  %b Email:
+  = "#{@form[:email]}"
+  %br
+  %b Job Role:
+  = "#{@form[:role]}"
+  %br
+  %b Job Title:
+  = "#{@form[:job_title]}"
+
+%p
+  %b School State:
+  = "#{@form[:school_state]}"
+  %br
+  - if @form[:school_district]
```

Comment:
What happens if they don't have a school district, school name, or message? Can we put `School District: None`?

Reply:
Right now if there is no district/name/message, those lines don't show up at all. I can change it to say "None" instead though if that's better!

Reply:
I'd like to do that so it's clearer. That's what I'm currently doing in my mails. I think they might have more questions like "what is the school? where'd it go?" if it doesn't show up at all.

Reply:
Got it- fixed!